### PR TITLE
Throwing a 404

### PR DESCRIPTION
**Pull request by zoe-edwards**
I use this all the time and I can never remember it! Instructions for how to use the new-style 404.

### CODE REVIEW

**Comment by zoe-edwards** [comment]
Just realised I sound like a Victorian. I guess that’s what comes of watching Downton Abbey before programming.

File: general/404.html [hunk 1]
```diff
@@ -60,6 +60,16 @@
...
 			
+			<h3 id="throw">Throwing a 404</h3>
+			
+			<p>
+				There may be times when one needs to throw a 404 error, such as when handling the routing yourself. 
+				This is simply done by throwing a <code>Request404Exception</code>. Fuel will <code>exit</code> for 
+				you once it has run the 404 page.
+			</p>
+			
+			<pre class="php"><code>throw new Request404Exception();</code></pre>
```
Comment (philsturgeon) [inline general/404.html]: If I was going to be a total douche I could say that you don't need to have the () at all.

**Comment by zoe-edwards** [comment]
Boosh.

**Comment by philsturgeon** [comment]
Bam!

**Pull request merged**